service/parameter.go

```diff
@@ -22,5 +22,5 @@ type Parameter struct {
 	Repeated bool `hash:"name:6"`
 
 	// Definition of the structure of the object when the type is object
-	Object []*Parameter `hash:"name:7"`
+	Object []*Parameter `hash:"name:7" validate:"unique,dive,required"`
```

Comment:
why this one is unique?
does it actually works?

Reply:
it won't be used here, but shouldn't exist